projects/addon-editor/components/editor-new/editor-new.component.ts

```diff
@@ -135,4 +143,11 @@ export class TuiEditorNewComponent
     private get hasValue(): boolean {
         return !!this.value;
     }
+
+    private getMarkedLinkBeforeSelectClosest() {
+        const [link] = this.editor?.state.tr.selection.$anchor.marks() ?? [];
```

Comment:
Maybe we should use `||` when there is no difference, just so transpiled bundle is smaller?

Reply:
fixed